src/core/sys/linux/sys/time.d

```diff
@@ -0,0 +1,72 @@
+/**
+ * D header file for POSIX.
+ *
+ * Copyright: Copyright Sociomantic Labs GmbH.
+ * License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors:   Leandro Lucarella
+ */
+
+/*          Copyright Sociomantic Labs GmbH.
+ * Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ */
+module core.sys.linux.sys.time;
+
+private import core.sys.linux.config;
+public import core.sys.posix.sys.types;  // timeval
+
+version (linux)
+else static assert(false, "Unsupported platform");
+
+version( CRuntime_Glibc )
+else static assert(false, "Unsupported platform");
```

Comment:
Mmm, it's very weird to have:
```d
version (linux) version (CRuntime_Glibc)
    public import core.sys.linux.sys.time;
```
At that file we don't really know about restrictions on that particular module and glibc runtime.

I see 3 possible solutions:
1. Ignore the `CRuntime_Glibc` completely and just hope nobody uses linux with a different (and incompatible) runtime.
2. Create a `core.sys.glibc.sys.time` and make it only versioned with `CRuntime_Glibc` (not mention linux in there and hope all glibc implementations in different platforms are the same).
3. Move the code to `core.sys.posix.sys.time` and put it versioned with both `CRuntime_Glibc` and `linux`, but then that's not really posix (which makes me think that, even when in C one would do a regular `#include <sys/time.h>`, maybe the automatic public import in `core.sys.posix.sys.time` shouldn't be done if there is some guarantee that if you stay in `core.sys.posix` your code won't call non-standard functions.

And I don't like any of them 100% :sweat: :gun: